feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/network/subscan/StakingApi.kt

```diff
@@ -3,24 +3,28 @@ package jp.co.soramitsu.feature_staking_impl.data.network.subscan
 import jp.co.soramitsu.common.data.network.subquery.SubQueryResponse
 import jp.co.soramitsu.common.data.network.subquery.SumRewardResponse
 import jp.co.soramitsu.common.data.network.subscan.SubscanResponse
+import jp.co.soramitsu.feature_staking_impl.BuildConfig
 import jp.co.soramitsu.feature_staking_impl.data.network.subscan.request.ExtrinsicHistoryRequest
 import jp.co.soramitsu.feature_staking_impl.data.network.subscan.request.StakingRewardRequest
 import jp.co.soramitsu.feature_staking_impl.data.network.subscan.request.StakingSumRewardRequest
 import jp.co.soramitsu.feature_staking_impl.data.network.subscan.response.ExtrinsicHistory
 import jp.co.soramitsu.feature_staking_impl.data.network.subscan.response.StakingRewardHistory
 import retrofit2.http.Body
+import retrofit2.http.Headers
 import retrofit2.http.POST
 import retrofit2.http.Path
 
 interface StakingApi {
 
-    @POST("//{subDomain}.subscan.io/api/scan/extrinsics")
+    @Headers("x-api-key: ${BuildConfig.SUBSCAN_API_KEY}")
```

Comment:
Probably we can configure interceptor with this header to not to repeat it over and over
But it is minor thing, for now it is fine